horovod/common/operations.cc

```diff
@@ -75,6 +108,63 @@ namespace horovod {
 namespace common {
 
 namespace {
+    
+#if HAVE_MLSL
+#define COMMUNICATOR_TABLE_MAX 1000
+#define NULL_VAL nullptr
+
+#define ERROR 0
+#define INFO  1
+#define DEBUG 2
+#define TRACE 3
+
+#define GET_TID() syscall(SYS_gettid)
+
+#define MLSL_LOG(log_lvl, fmt, ...)					\
```

Comment:
There's another PR that adds logging - #671, we should switch to use it once it's merged.

Reply:
@alsrgv We can change our logging system to the one from the PR you referred to anytime (after it will be merged). Does this mean that our PR has to wait until [https://github.com/uber/horovod/pull/671](url) is merged (as far as I can see there are still some unsuccessful checks)?

Reply:
#671 has been merged, please incorporate its functionality.